### ISSUE TITLE
Create domain job fails after 300 seconds although domain is created

### ISSUE THREAD

**Issue by trobert57** [oracle/weblogic-kubernetes-operator#109]
Running  ./create-domain-job.sh –i create-domain-job-inputs.yaml terminates after 5 minutes with error

=========================================================
The create domain job is not showing status completed after waiting 300 seconds
Check the log output for errors
The domain will be created using the script /u01/weblogic/create-domain-script.sh

Initializing WebLogic Scripting Tool (WLST) ...

Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands

domain_path        : [/shared/domain/my_wls_domain]
domain_name        : [my_wls_domain]
admin_username     : [weblogic]
admin_port         : [7001]
cluster_name       : [cluster-1]
server_port        : [8001]
managed server name is managed-server1
managed server name is managed-server2
Domain Created
[ERROR] Exiting due to failure

=========================================================

No weblogic docker container is running then, but one is in stopped state. It's logfile says:

=========================================================
The domain will be created using the script /u01/weblogic/create-domain-script.sh

Initializing WebLogic Scripting Tool (WLST) ...

Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands

domain_path        : [/shared/domain/my_wls_domain]
domain_name        : [my_wls_domain]
admin_username     : [weblogic]
admin_port         : [7001]
cluster_name       : [cluster-1]
server_port        : [8001]
managed server name is managed-server1
managed server name is managed-server2
Domain Created
Domain Updated
Create boot.properties files for admin and managed servers
Done

Exiting WebLogic Scripting Tool.

Successfully Completed

=========================================================

So - what might be going wrong here?

**Comment by rjeberhard** [comment]
Just to clarify, I believe you are saying that you've run the domain creation script and it appears to have created the domain but then exited with an error after creating the domain, but before it finished.  Is this correct?  Can you tell us what other resources were created in the target namespace?

**Comment by trobert57** [comment]
Actually the create-domain-job.sh skript exited with error, saying: 
"The create domain job is not showing status completed after waiting 300 seconds"
The logfile of the pod says: 
"Exiting WebLogic Scripting Tool.

Successfully Completed"
But only AdminServer is completed, no Managed Server.
The resources in the target namespace for the domain (-n wlsdomain1) are:
Pods: domain-mydomain1-job-b4jbr
Jobs: domain-mydomain1-job
Config Maps: domain-mydomain1-scripts
Persistent Volume Claims: mydomain1-pv001-claim
3 Secrets: docker-store-secret, mydomain1-weblogic-credentials, default-token-mwbsk

I do have a docker container on one node, which is in stopped state, with name "k8s_domain-job_domain-mydomain1-job-b4jbr_wlsdomain1_de8e4135-18a7-11e8-9114-020017027e48_0". Its logfile says that the domain, including two managed servers have been created successfully. 

Maybe a timing issue?!

**Comment by trobert57** [comment]
I ran it again today. The create-domain-job.sh again fails after 5 minutes with:
"The create domain job is not showing status completed after waiting 300 seconds"
But this time I ran a "docker logs -f <id>" on the container creating the domain. This container continues running even after the create-domain-job.sh failed and ~ 2 minutes later the log output shows: 
"Exiting WebLogic Scripting Tool.

Successfully Completed"
After modifying the create script to iterate max=20, it finished successfully after 17 (!) iterations.
So it works now ...
